src/components/SignUp/Account.vue

```diff
@@ -3,45 +3,57 @@
       <div class="account-field">
         <v-text-field
           class="account-inputs"
+          :rules="firstNameRules"
```

Comment:
**This goes for every field with rules:** whenever there's an error the hint should be binded to the error color